addon/components/collapsable-facet/template.hbs

```diff
@@ -9,25 +9,30 @@
   </div>
   <div class="{{options.groupListWrapperClass}} collapse{{if open " in"}}" id="{{facetId}}">
     <div class="{{options.groupListClass}}">
-    {{#each facet.terms as |term|}}
-      <div class="term">
-        <span class="term">
-        {{#if facet.link}}
-          <a href="#" class="{{if term.selected "active"}}" {{action "toggleFacet" term}}>{{if term.display term.display term.value}}</a>
-        {{else if facet.checkbox}}
-          <label>
-          {{input type="checkbox" checked=term.selected click=(action "toggleFacet" term)}}
-            {{if term.display term.display term.value}}
-            </label>
+      {{#each facet.terms as |term|}}
+        <div class="term">
+          <span class="term">
+          {{#if facet.link}}
+            <a href="#" class="{{if term.selected "active"}}" {{action "toggleFacet" term}}>{{if term.display term.display term.value}}</a>
+          {{else if facet.checkbox}}
+            <label>
+            {{input type="checkbox" checked=term.selected click=(action "toggleFacet" term)}}
+              {{if term.display term.display term.value}}
+              </label>
+          {{else if facet.datetime}}
+            <div class="form-group form-inline">
+              <label for="filter-from-time">{{term.display}} </label>
+              <span class="{{if facet.hasError "has-error"}}">{{bs-datetimepicker date=term.value sideBySide=true change=(action "toggleFacet" term)}}</span>
```

Comment:
Ah, never mind - I didn't realize that you were passing in a Facet w/ 2 terms ( fromTerm and toTerm ) and rendering the range that way.  Clever!